src/settings.rs

```diff
@@ -85,6 +93,10 @@ impl TryFrom<InternalSettings> for Settings {
         if internal.sats_per_claim > MAX_SATS_PER_CLAIM {
             panic!("sats per claim is too high, max is {MAX_SATS_PER_CLAIM}");
         }
+        if (internal.sats_per_claim > Amount::ONE_BTC) == false {
+            panic!("sats per claim is too low, must be greater than 1 BTC");
+        }
```

Comment:
It seems a little off to have this check (and the preceding one) panic instead of returning an error to be handled by the caller's `expect`.

Reply:
Updated to return an error.